extensions/amp-story/1.0/amp-story.js

```diff
@@ -570,6 +571,34 @@ export class AmpStory extends AMP.BaseElement {
     this.storeService_.dispatch(Action.SET_PAGE_IDS, pageIds);
   }
 
+  /**
+   * Initializes interactives by deduplicating their IDs and calling initializeState().
+   * @private
+   */
+  initializeInteractives_() {
+    const interactiveEls = this.element.querySelectorAll(
+      'amp-story-interactive-binary-poll, amp-story-interactive-poll, amp-story-interactive-quiz'
+    );
+    const interactiveIds = Array.prototype.map.call(
```

Comment:
Plz add an explicit early return when there's no interactive components, and use the `toArray()` helper instead of the `Array.prototype.map`